doc/api/source/index.html.md.erb

```diff
@@ -16,21 +16,54 @@ title: About
 
 ## What does this API do?
 
-This API provides access to metrics about the performance of GOV.UK content.
-Each metric is a time series, so you can see how the content performance has
-changed over time.
+This API provides access to time series metrics data about the performance of GOV.UK content. Examples of metrics include unique page views and number of PDF attachments. Refer to the [getting started section][] for a complete list of metrics.
 
-This API does not expose the content itself. For that you should use the [GOV.UK Content API][].
+This API does not expose the content itself. Use the [GOV.UK Content API][] to expose the content.
 
-Not all content is available. For example, content that is generated
-dynamically (such as through a search) is not available.
+## Rate limiting
+
+There is a maximum limit of 10 requests per second per client. All requests over this limit will not be processed, and you will receive a response with `HTTP status code 429 (too many requests)`.
+
+Your application should handle this exception by retrying the request at a later time.
+
+[Contact support][] if you want to increase this limit.
+
+## Version
+
+GOV.UK Content Performance API is currently in alpha and is likely to change between versions.
+
+You may use the API and build applications that use it, but live services should not currently rely on the API or the data it provides.
+
+As we learn from feedback we may make changes to the software. We will communicate those using our mailing list.
 
 ## Before you start
 
 ### Register an API key
-Usage of GOV.UK Content Performance API currently requires [authentication](#authentication-amp-authorisation) using an OAuth2 bearer token. Please [contact the team](#support) first if you are interested in using the API.
+
+Using GOV.UK Content Performance API currently requires [authentication][] using an OAuth2 bearer token.
+
+#### GOV.UK users:
+
+GOV.UK users can configure admin access by:
+
+1. Going to [GOV.UK Signon][]
+2. Clicking API Users
+3. Clicking Create API user button
+4. Creating API user
+5. Copying the bearer token
+
+#### GDS users:
+
+Should contact us through #govuk-developers on [GOV.UK Slack][].
+
+#### Government users:
+
+Should contact us through Zendesk.
+
+Please [contact the team] first if you are interested in using the API.
```

Comment:
Is this line still needed or can we combine with the previous sentence?

Reply:
Agree - Contact the GOV.UK Content Performance API team through Zendesk (link)

Reply:
Done